terraform/hcl/applicable/req_sw_terraform_aws_cloudfront_tls_version.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_cloudfront_distribution" "vulnerable_example" {
```

Comment:
[comment]: <> (FrogbotReview Iac 1 1 23 2 "vulnerable_example" /terraform/hcl/applicable/req_sw_terraform_aws_cloudfront_tls_version.tf)

## ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableLowSeverity.png)<br>     low Infrastructure as Code Vulnerability 🛠️
	
Finding: Deprecated TLS version was detected

### 👇 Details

Resource `aws_cloudfront_distribution` should have `viewer_certificate.minimum_protocol_version = 'TLSv1.2*'` or `viewer_certificate.minimum_protocol_version = 'TLSv1.3*'`

Vulnerable example -
```
resource "aws_cloudfront_distribution" "vulnerable_example" {
  viewer_certificate {
    minimum_protocol_version  = "TLSv1"
  }
}
```

Secure example -
```
resource "aws_cloudfront_distribution" "secure_example" {
  viewer_certificate {
    minimum_protocol_version  = "TLSv1.2_2019"
  }
}

```	

[[🐸 JFrog Frogbot]](https://github.com/jfrog/frogbot#readme)